xc7/archs/artix7/CMakeLists.txt

```diff
@@ -1,9 +1,11 @@
 project_xray_prepare_database(
-  PART artix7
+  ARCH artix7
+  PART xc7a35tcpg236-1
```

Comment:
So while this does accommodate the 200T, this causes multiple copies of the arch/rrgraph/lookahead to be computed for graphs that are identical.  Keep in mind that the parts in the 35T share a graph, but have different pinouts (e.g. packaging).  It seems like a mistake for us to no re-use the arch/rrgraph/lookahead for parts that share a graph, but have different pinouts.  @mithro / @kgugala  How much do we care?